package.json

```
@@ -72,8 +72,8 @@
     "precommit": "ts-node ./scripts/precommit.ts"
   },
   "engines": {
-    "vscode": "^1.94.2",
-    "node": ">=20.9.0",
+    "vscode": "^1.101.1",
+    "node": "^20.19.0 || ^22.12.0 || >= 23.0.0",
```

23.x is EOL, no real point in adding it now
```suggestion
    "node": "^20.19.0 || ^22.12.0 || >= 24.7.0",
```